### PR TITLE
[Term Entry] Kotlin Math.cosh()

### PR DESCRIPTION
**Pull request by Kadif-Ravik**
### Description

Kotlin math function cosh entry

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by Kadif-Ravik** [comment]
@SSwiniarski

File: content/kotlin/concepts/math-methods/terms/cosh/cosh.md
Comment (SSwiniarski) [inline content/kotlin/concepts/math-methods/terms/cosh/cosh.md:3]:
```suggestion
Description: 'Computes the hyperbolic cosine of an angle.'
```

File: content/kotlin/concepts/math-methods/terms/cosh/cosh.md
Comment (SSwiniarski) [inline content/kotlin/concepts/math-methods/terms/cosh/cosh.md:16]:
```suggestion
The **`cosh()`** method in Kotlin's `math` class computes the hyperbolic cosine of a angle given in radians.
```

File: content/kotlin/concepts/math-methods/terms/cosh/cosh.md
Comment (SSwiniarski) [inline content/kotlin/concepts/math-methods/terms/cosh/cosh.md:27]:
```suggestion
Special cases:

```

File: content/kotlin/concepts/math-methods/terms/cosh/cosh.md
Comment (SSwiniarski) [inline content/kotlin/concepts/math-methods/terms/cosh/cosh.md:46]:
```suggestion
This example will output:
```

Comment (Kadif-Ravik) [reply content/kotlin/concepts/math-methods/terms/cosh/cosh.md:46]: @SSwiniarski Thank you.

**Review by SSwiniarski — APPROVED** [review]
@Kadif-Ravik looks good for a second review.

**Comment by Kadif-Ravik** [comment]
@SSwiniarski whats next

**Comment by SSwiniarski** [comment]
@Kadif-Ravik Another maintainer will review this and possibly ask for edits before merging.

**Review by caupolicandiaz — APPROVED** [review]
@Kadif-Ravik @SSwiniarski I made a few direct edits, looks good to merge

**Comment by Kadif-Ravik** [comment]
@SSwiniarski Thanks

**Pull request merged**
